contracts/src/Gateway.sol

```diff
@@ -428,9 +428,11 @@ contract Gateway is IGatewayBase, IGatewayV1, IGatewayV2, IInitializable, IUpgra
             revert IGatewayBase.InvalidProof();
         }
 
-        // Dispatch the message payload
+        // Dispatch the message payload. The boolean returned indicates whether all commands succeeded.
         bool success = v2_dispatch(message);
 
+        // Emit the event with a success value "true" if all commands successfully executed, otherwise "false"
+        // if all or some of the commands failed.
```

Comment:
I think this documentation should be moved to the the comment for `InboundMessageDespatched` in `IGateway.sol`.

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/1469/commits/37a2a4352403831e0dd149f66c7f85aac9c188ae.